index.html

```diff
@@ -0,0 +1,222 @@
+<!dOCTYPE html>
+<html lang="zxx">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Smashing Magazine design teardown</title>
+    <link href="https://fonts.googleapis.com/css?family=Acme&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Patua+One&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="assets/stylesheets/reset.css">
+    <link rel="stylesheet" href="assets/stylesheets/main.css">
+</head>
+
+<body>
+    <header class="w6">
+        <div>
+            <h1>Smashing <br>
+                Magazine</h1>
+        </div>
+        <nav>
+            <ul>
+                <li>
```

Comment:
As was told before, each one of the seven elements here in the navbar must be a container. Put each one inside a `div`